src/crawler.cls.php

```diff
@@ -255,10 +281,17 @@ public static function async_handler($manually_run = false)
 	 */
 	public static function start($manually_run = false)
 	{
+		$crawler_is_in_time = self::_crawler_in_schedule_time();
```

Comment:
This should be done after L289, also considering the context, share the same instance as `self::cls()->_crawl_data($manually_run);`

Reply:
Is this better?